lib/page-generate.jsx

```diff
@@ -100,9 +101,10 @@ function run(location, el) {
     we will be using the value there otherwise we will default to
     what was assigned above.
   */
-  supportedLocale = Object.keys(locales).indexOf(navigator.language) !== -1;
-  currentLocale = supportedLocale ? navigator.language : currentLocale;
+  currentLocale = window.location.pathname.split('/')[1];
   messages = locales[currentLocale] || messages;
+  // Load React's data for the current locale, but if it's not available for that specific country, load the general language data
+  addLocaleData(window.ReactIntlLocaleData[currentLocale] || window.ReactIntlLocaleData[currentLocale.split('-')[0]]);
```

Comment:
I'm not sure I understand the benefit of this approach- right now I'm just pulling in the one language to memory, which is the only one that's loaded in index.js's script tag. What else would be in window.ReactIntlLocaleData at this point? (Also, I needed to do the split because `bn-BD` isn't a recognized locale by react-intl and it defaulted back to a broken 'en')